openaerostruct/aerodynamics/tests/test_geometry.py

```diff
@@ -97,7 +97,7 @@ def test_derivs_wetted(self):
         prob.run_model()
 
         check = prob.check_partials(compact_print=True, step=1e-7)
-        assert_check_partials(check, atol=5e-5, rtol=1e-5)
+        assert_check_partials(check, atol=7.5e-5, rtol=1e-5)
```

Comment:
Do we need a better step to get more accurate FD values? If we don't need to relax the tolerance we should try not to

Reply:
This was something I was seeing locally on my computer, but since it's not a problem in the build I'll undo it. For some reason when I run `python -m pytest` on the full repo, this check fails. But when I run only this test (`python -m pytest openaerostruct/aerodynamics/tests/test_geometry.py`), it passes. I can't explain it

Reply:
Yeah no idea. Probably different numpy versions or something.